api/controller/ordercontroller.js

```diff
@@ -73,14 +93,17 @@ export const createOrderItems = (req, res) => {
     });
   }
   const { itemName, itemPrice, quantity } = req.body;
+  const newItemName = itemName.trim();
+  const newItemPrice = itemPrice;
+  const newQuantity = quantity;
   const newOrderItems = {
     id: orderItems.length + 1,
     orderId,
-    itemName,
-    itemPrice: Number(itemPrice),
-    quantity: Number(quantity),
+    itemName: newItemName,
+    itemPrice: Number(newItemPrice),
+    quantity: Number(newQuantity),
   };
-  if ((!itemName) || (!itemPrice) || (!quantity)) {
+  if ((!newItemName) || (!newItemPrice) || (!newQuantity)) {
```

Comment:
This seems like a validation that is currently being done in the middleware, why is it here again?

Reply:
i have removed it